rush.json

```diff
@@ -78,6 +78,12 @@
       "reviewCategory": "tests",
       "shouldPublish": false
     },
+    {
+      "packageName": "rush-stack-library-test",
+      "projectFolder": "build-tests/rush-stack-library-test",
+      "reviewCategory": "tests",
+      "shouldPublish": false
+    },
```

Comment:
Add these to the root readme.

Reply:
I'm not quite ready to advertise them yet.

---
In reply to: [179332136](https://github.com/Microsoft/web-build-tools/pull/602#discussion_r179332136) [](ancestors = 179332136)